instana/util/traceutils.py

```diff
@@ -30,3 +30,16 @@ def get_active_tracer():
         # Do not try to log this with instana, as there is no active tracer and there will be an infinite loop at least
         # for PY2
         return None
+
+
+def get_tracer_tuple():
+    active_tracer = get_active_tracer()
+    if get_active_tracer():
+        return active_tracer, active_tracer.active_span, active_tracer.active_span.operation_name
+    elif agent.options.allow_exit_as_root:
+        return tracer, None, None
```

Comment:
Can we explicitly return a tuple in this function for better readability?
```suggestion
        return (tracer, None, None)
```

Reply:
Done